crates/bevy_reflect/src/func/return_type.rs

```diff
@@ -6,8 +6,6 @@ use crate::PartialReflect;
 /// [`DynamicFunctionMut`]: crate::func::DynamicFunctionMut
 #[derive(Debug)]
 pub enum Return<'a> {
-    /// The function returns nothing (i.e. it returns `()`).
-    Unit,
     /// The function returns an owned value.
```

Comment:
I think we need to warn the user that this value can be "nothing" (Like Return::Unit says above in the docs)

Reply:
I added something to let users know.

I am not the best at writing documentation, so if it should be written in a different way, let me know,

Reply:
@hooded-shrimp gave a suggestion :)